### DISCUSSION TITLE
Offline?

### DISCUSSION THREAD

**Discussion by iamthezio** [w2c/letsencrypt-esxi#15] [General]
Installing a new ESXi 7 and would like to try. Is this github offline?
I get time out when attempting to pull.

wget -O /tmp/w2c-letsencrypt-esxi.vib https://github.com/w2c/letsencrypt-esxi/releases/latest/download/w2c-letsencrypt-es
xi.vib

**Comment by Churro** [comment]
The [link](https://github.com/w2c/letsencrypt-esxi/releases/latest/download/w2c-letsencrypt-esxi.vib) works fine. Please check your firewall settings or upload the VIB file manually.
